Logcollector integration tests T0: Check location values (#1301)
wazuh/wazuh-qa: docs/tests/integration/test_logcollector/test_location/index.md

# Test location

## Overview 

> These tests check if the introduced configuration works as expected for valid values and if the logcollector 
> response for analyzing requests works properly.

Reviewer: ```suggestion
Check if the specified location values work as expected. 
```

Author: Fixed in 8e8e84a2a739ed54d49446ac6e5db42ccb2d59dd

## Objective

> Confirm that the different options for logcollector configuration work and are correctly loaded.

Reviewer: ```suggestion
Confirm that the different location option values for logcollector configuration work and are correctly loaded.
```

Author: Fixed in 8e8e84a2a739ed54d49446ac6e5db42ccb2d59dd

# Test location

## Overview 

These tests check if the introduced configuration works as expected for valid values and if the logcollector 
response for analyzing requests works properly.

## Objective

Confirm that the different options for logcollector configuration work and are correctly loaded.

## General info

|Tier | Number of tests | Time spent |
|:--:|:--:|:--:|
| 0 | 21 | 42.20s |

## List of configuration tests

- **[Test location](test_location.md)**: Check if logcollector works using different `location` values.

- **[Test exclude](test_exclude.md)**: Check if logcollector works using different `location` and `exclude` values.

> 
> 
> 
> 

Reviewer 2: ```suggestion
```

Author: Fixed in https://github.com/wazuh/wazuh-qa/commit/8dd2eb232ec5e6ed9c8b14ba1437da2b51a77b63
